README.md

```diff
@@ -186,7 +186,7 @@ The following parameters are available for the hiera class:
   This is useful for configuring backend-specific parameters.  
   Default: `''`
 * `keysdir`
-  Directory for hiera to manage for eyaml keys.
+  Directory for hiera to manage for eyaml keys. This is the path where /keys should be managed not the full path to the /keys directory.
```

Comment:
@smbambling I was also thinking that the `hiera::keysdir` parameter would be the full path to keys/, like `keysdir => /etc/puppet/eyaml_keys` or whatever. What do you think?